spec/models/comment_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  describe 'Validations for the comment model' do
+    before(:each) do
+      @comment = Comment.new(text: 'First comment', author_id: 2, post_id: 1)
+    end
+
+    before { @comment.save }
+
+    it 'if author_id is present' do
+      @comment.text = nil
+      expect(@comment).to_not be_valid
+    end
+
+    it 'if author_id is present' do
+      @comment.author_id = 'a'
+      expect(@comment).to_not be_valid
+    end
+
+    it 'if post_id is present' do
+      @comment.post_id = nil
+      expect(@comment).to_not be_valid
+    end
+  end
```

Comment:
- [ ] As required for the project for any custom methods in the models to have a corresponding unit test, kindly ensure that you add test on the `update_comments_counter` method

- [ ] Once you have added the above change you should make sure that all your tests are still passing. But good job :+1: :+1: